### ISSUE TITLE
Answers from BioGrid

### ISSUE THREAD

**Issue by manulera** [pombase/curation#3295]
> * Overexpression of gene X is lethal or causes growth defect, but a mutation/deletion/overexpression of gene Y rescues that phenotype. Dosage Rescue and Synthetic rescue specify that the gene X can be only mutated/deleted. Phenotypic Suppression excludes lethality/ growth defect phenotypes.
> * Overexpression of gene X causes lethality or growth defect in a strain that overexpresses gene Y. Dosage Growth Defect and Dosage Lethality specify that gene Y can only be mutated/deleted. 

We consider the overexpression of gene X to be a genetic perturbation and capture this interaction as Synthetic rescue if gene Y is mutated or deleted and restores growth/viability or Dosage Rescue if gene Y is overexpressed and restores growth/viability. Looking at our definition though I completely agree that the terminology used (mutated/deleted) makes it sound like we do not include these. I will discuss with the other curators at our next meeting the possibility of modifying the definition so that it more clearly reflects our curation practices. 

> Can knockdowns / reduced expression of genes be used to stablish genetic interactions? Should a knockdown be treated as a deletion/mutation? I imagine that for certain organisms where RNA silencing is common some genetic interactions must have been annotated from siRNA experiments.

Yes, we also treat knockdowns as deletion/mutation. I think we may change that wording from deletion/mutation to "genetic perturbation" and add a specific note that it can refer to engineered changes in expression level in addition to deletions and mutations.

**Comment by manulera** [comment]
Hello @ValWood, writing here so that we can discuss it. I think I would agree for the knockdowns, not sure about overexpressions.

I think probably the issue is that these terms are from the early days of genetics, so those interactions  `deleted/mutated` probably referred to the loss of function of a given gene.

I think when people think of synthetic rescue they would think of double loss of function rescuing single loss of function which causes growth deficiency / lethality. For example, double deletion rescuing single deletion, or deletion rescuing thermosensitive mutant. With this perspective, I can see how deletion and knockdown can be used interchangeable, but using overexpression as background would mean loss of function of a gene rescues excess function of another gene.

Same for Dosage rescue, where you probably expect "more of X rescues less of Y".

Not sure I have a good suggestion on what to do. Maybe we make an internal decission and we inforce it when we revise community curation sessions?

**Comment by manulera** [comment]
A followup question that I sent as well:

> Dear BioGrid colleagues,
> 
> I am writing from Pombase. We were wondering if synthetic phenotypes are > considered genetic interactions and whether you have some recommendations to > annotate them.
> 
> I would define a synthetic phenotype as double mutation of X and Y causes a > phenotype that does not exist if X or Y alone are mutated.
> 
> I guess they could be covered by the "negative interaction", but these are > "reserved for high or low throughput studies with scores"
> 
> Kind regards,
> Manu

Their answer:

Hello Manu,

Thanks for reaching out again with your questions regarding our genetic interactions experimental systems.

It might be helpful if I provide a very brief background on our genetic interactions. Originally, we began capturing genetic interactions as part of our comprehensive Saccharomyces cerevisiae curation and we adopted the experimental systems that were in use at SGD at the time. As our curation has continued and expanded to other organisms we have run across cases where the original experimental systems and definitions were not a great fit. 

Your example of a synthetic phenotype (non-growth or viability) that does not exist in the single mutants is a good example. When we came across examples like this in the literature we decided to use "phenotypic enhancement" as a best fit option. Although it's not ideal as it implies the presence of the phenotype in one of the starting single mutants. Much like the other issue you brought to our attention it would be a good idea for us to update the definition on our guidelines page to reflect this curation standard and I will bring that up at our next curator meeting.

A few years ago one of our curators collaborated with Chris Grove at the Genome Alliance on a new genetic interaction system that would be more comprehensive, be flexible enough to handle a wide variety of organisms and would allow us to address some of these issues, including the issue of single starting mutant phenotypes in relation to the double mutants. Unfortunately, some of the MODs were reluctant to implement it due to the work that would be involved in changing over their curation. At this point, I believe our Saccharomyces genetic interactions would be too time-consuming to try and recurate to different definitions as well. Chris was able to add the BioGRID terms into the PSI-MI ontology though. If you're interested I could put you in touch with him for more information.

For now I will definitely talk to the other curators to modify our definition of "Phenotypic Enhancement" to make it clear that we also use it when a synthetic phenotype (that is not specifically growth or lethality) is seen in XY mutants where there is no phenotype in the single X or Y mutant.

Thank you!
Jenn
